test/blocks/fragment/fragment.test.js

```diff
@@ -44,4 +44,17 @@ describe('Fragments', () => {
     expect(document.querySelector('h4')).to.exist;
     expect(window.lana.log.args[2][0]).to.equal('ERROR: Fragment Circular Reference loading http://localhost:2000/test/blocks/fragment/mocks/fragments/frag-a');
   });
+
+  it('Inlines fragments inside a block', async () => {
+    const marquee = document.querySelector('.marquee-section');
+    await loadArea(marquee);
+    // await waitForUpdate(marquee);
```

Comment:
If this is no longer needed, it should be removed, as well as the `waitForUpdate` import, so ESLint stops complaining about it.